### PR TITLE
UI issue fix

### PR DESCRIPTION
**Pull request by Jayakrishnan-MS**

### CODE REVIEW

File: src/ui/CustomStyleEditor.js [hunk 1]
```diff
@@ -1118,7 +1122,7 @@ class CustomStyleEditor extends React.PureComponent<any, any> {
                       <input
                         checked={this.state.styles.boldNumbering}
                         className="molsp-chkboldnumbering"
-                        disabled={this.state.styles.hasNumbering ? false : true}
+                        disabled={!(this.state.styles.hasNumbering) || (this.state.styles.styleLevel === 'None' || this.state.styles.styleLevel === undefined) ? true : false}
```
Comment (Tezra) [inline src/ui/CustomStyleEditor.js:1125]: Ambiguity on the '!' intent looks like a possible mistake. The '!' should either be inside the bubble, or the bubble should cover more than 1 expression.

File: src/ui/CustomStyleEditor.js [hunk 2]
```diff
@@ -1128,15 +1132,15 @@ class CustomStyleEditor extends React.PureComponent<any, any> {
...
+                        disabled={(this.state.styles.styleLevel === 'None' || this.state.styles.styleLevel === undefined) ? true : false}
                         onChange={this.handleBulletPoints.bind(this)}
                         type="checkbox"
                       />
                       Bullet&nbsp;
                       <span>
                         <select
                           className="molsp-fontstyle"
-                          disabled={this.state.styles.hasBullet ? false : true}
+                          disabled={!(this.state.styles.hasBullet) || (this.state.styles.styleLevel === 'None' || this.state.styles.styleLevel === undefined) ? true : false}
```
Comment (Tezra) [inline src/ui/CustomStyleEditor.js:1143]: Ambiguity on the '!' intent looks like a possible mistake. The '!' should either be inside the bubble, or the bubble should cover more than 1 expression.

Comment (Tezra) [inline src/ui/CustomStyleEditor.js:1143]: This Boolean expression is used multiple times. It should be extracted out to a function.

File: src/ui/CustomStyleEditor.js [hunk 3]
```diff
@@ -887,7 +891,7 @@ class CustomStyleEditor extends React.PureComponent<any, any> {
                       disabled={this.state.styles.boldPartial ? false : true}
                       name="boldscentence"
                       onChange={this.onScentenceRadioChanged.bind(this)}
-                      style={{ marginLeft: '20px' }}
+                      style={{ marginLeft: '21px' }}
```
Comment (Tezra) [inline src/ui/CustomStyleEditor.js:894]: Is 21 a font size? This looks like it should be in em instead of px.

File: src/ui/CustomStyleEditor.js [hunk 4]
```diff
@@ -1428,6 +1435,7 @@ class CustomStyleEditor extends React.PureComponent<any, any> {
     const hiddenDiv = document.getElementById('nextStyle');
     if (hiddenDiv && hiddenDiv.style) {
       hiddenDiv.style.display = display;
+      hiddenDiv.style.marginBottom = '-7px';
```
Comment (Tezra) [inline src/ui/CustomStyleEditor.js:1438]: Is 7 a fraction of a font size? This looks like it should be in em instead of px.

**Review by Tezra — CHANGES_REQUESTED** [review]

File: src/ui/CustomStyleEditor.js (hunk 2 above)
Comment (rohanrjohn) [reply src/ui/CustomStyleEditor.js]: Have changed this to a function.

File: src/ui/CustomStyleEditor.js (hunk 3 above)
Comment (rohanrjohn) [reply src/ui/CustomStyleEditor.js:909]: This changes the position of the radio button. I have changed the value, so that it looks organized.

**Review by Tezra — APPROVED** [review]

**Review by rodgersgb — APPROVED** [review]

**Pull request merged**